client/src/common/components/Members/components/MemberDetails.jsx

```diff
@@ -243,9 +224,17 @@ class MemberDetails extends PureComponent {
   };
 
   renderDetails = () => {
-    const { isMemberInfoVisible, isOwnerInfoVisible } = this.state;
+    const { isMemberInfoVisible, isOwnerInfoVisible, pending } = this.state;
     const { isGuest, isPrivate: privateList, route } = this.props;
 
+    if (pending) {
```

Comment:
why not ternary here as everywhere??

Reply:
There was a ternary, but it was removed to add quick return